patches/0001-trace-Make-code-compatible-for-older-a.patch

```diff
@@ -7,16 +7,16 @@ After changing assign_str to one argument in KV6.10, backporting to
 older version need to change parameters with in trace definition.
 Go back to old handling.
 
-Error:
---------------------------------------------------------------------------
-error: macro "__assign_str" requires 2 arguments, but only 1 given
---------------------------------------------------------------------------
+Reference-id:
+	2c92ca849fcc
+	tracing/treewide: Remove second parameter of __assign_str()
 
-Reference-id: 2c92ca849fcc tracing/treewide: Remove second parameter of
-__assign_str()
+v1: Rebase
```

Comment:
dont need rebase and hunk for single update. hunk should not be part of first commit. lets remove them